test/producer.ts

```diff
@@ -221,6 +222,39 @@ describe('Producer', () => {
     await rejects(producer.send(['foo']), errMessage);
   });
 
+  it('returns an AWS response', async function () {
```

Comment:
Could you please update this test with a more meaningful label and update it to arrow function with const instead of vars.

Please double-check it using npm run lint as I can see there are some eslint complaints!